components/typography/CardTitle.tsx

```diff
@@ -0,0 +1,9 @@
+import React, {FC} from "react";
+import {Typography} from "@mui/material";
+
+interface Props {
+    title: string;
+}
+
+export const CardTitle: FC<Props> = ({title}) =>
```

Comment:
Maybe this can be used as the TextField component, so it accepts something to render as a child